example/src/spa/view/header/header-view.js

```diff
@@ -0,0 +1,83 @@
+import './header.css';
+// eslint-disable-next-line no-unused-vars
+import Router from '../../router/router';
+import { Pages } from '../../router/pages';
+import View from '../../../classes/view/view';
+import ElementBuilder from '../../../classes/util/element/element-builder';
+
+const CssClasses = {
+    HEADER: 'header',
+    NAV: 'nav',
+    ITEM: 'nav-item',
+    ITEM_SELECTED: 'nav-item__selected',
+};
+const NamePages = {
+    INDEX: 'Главная',
+    SHOWCASE: 'Карточки',
+    ABOUT: 'О нас',
+};
+
+export default class HeaderView extends View {
+    /**
+     * @param {Router} router
+     */
+    constructor(router) {
+        super();
+
+        this.headerLinkElements = new Map();
+        this.htmlElement = this.createView(router);
+    }
+
+    /**
+     * @param {Router} router
+     */
+    createView(router) {
+        const headerBuilder = new ElementBuilder('header');
```

Comment:
If we need class without additional functionality, we would use
`const headerBuilder = new BaseComponent({tag: 'header', classNames: ['header']});`